### PR TITLE
updated cratedb branding for docs

### PR DESCRIPTION
**Pull request by jessicarose**
Docs updated to CrateDB branding, text

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,11 +4,11 @@
     :target: https://crate.io
 
 =================================
-CRATE: Put data to work. Simply.
+CRATEDB: Put data to work. Simply.
 =================================
```
Comment (chaudum) [inline README.rst]: title undeline too short.
needs to have at least same length as title itself. ([rst](http://docutils.sourceforge.net/docs/user/rst/quickref.html#section-structure))

please check all occurrences!

File: blackbox/docs/cli.txt [hunk 2]
```diff
@@ -59,21 +59,21 @@ Example::
 Signal Handling
 ===============
 
-The Crate process can handle the following signals.
+The CrateDB process can handle the following signals.
 
 +-----------+-------------------------------------------+
 | Signal    | Description                               |
```
Comment (chaudum) [inline blackbox/docs/cli.txt]: extend table width

File: blackbox/docs/udc.txt [hunk 3]
```diff
@@ -46,8 +46,13 @@ done with a HTTP GET.
 
 Admin UI tracking
 =================
+<<<<<<< Updated upstream
```
Comment (chaudum) [inline blackbox/docs/udc.txt]: resolve merge conflict!

File: blackbox/docs/plugins.txt [hunk 4]
```diff
@@ -18,10 +18,10 @@ See our `example crate plugin`_ for details about that.
 
 .. _plugins_crate_dep:
 
-Plugin Crate Dependency
+Plugin CrateDB Dependency
 =======================
```
Comment (chaudum) [inline blackbox/docs/plugins.txt]: title underline length

File: blackbox/docs/installation.txt [hunk 5]
```diff
@@ -73,12 +73,12 @@ on how to run Crate in a Docker container on the `docker-crate`_ Github reposito
 
 .. _install_production:
 
-Install Crate for Production
+Install CrateDB for Production
 ============================
```
Comment (chaudum) [inline blackbox/docs/installation.txt]: title underline length

File: blackbox/docs/hello.txt [hunk 6]
```diff
@@ -1,12 +1,12 @@
 .. highlight:: sh
 
 ===========
-Hello Crate
+Hello CrateDB
 ===========
```
Comment (chaudum) [inline blackbox/docs/hello.txt]: title underline length

**Comment by jessicarose** [comment]
Fixing now, sorry/thanks!

**Pull request closed without merging**